### PR TITLE
Ipfix pkt gen

### PR DESCRIPTION
**Pull request by sosaavedra**
Add script to generate and send IPFIX Messages.
It can simulate bursty traffic by using the double
of the specified rate speed. Additionally, it can
transmit at rate speeds below 10 Mbit/s. A CSV
report is generated with the number of packets,
flows and time.

### CODE REVIEW

File: examples/ipfix-generation.lua [hunk 1]
```diff
@@ -0,0 +1,247 @@
...
+-- IPFIX Packet Constants
+local PREV_HEADERS_LENGTH   = 42                          -- In octets
+local OBSERVATION_DOMAIN    = 67108864                    -- Observation Domain
+local SELECTOR_ALGORITHM    = 1                           -- Systematic count-based Sampling
+local SAMPLING_PKT_INTERVAL = 1                           -- Sampling Packet Interval
+local SAMPLING_PKT_SPACE    = 9999                        -- Sampling Packet Space
+local PROTO_ID              = { 6, 17 }                   -- Protocol Identifier
+local SRC_TRANS             = { 80, 100 }                 -- Source Transport Port
+local DST_TRANS             = { 180, 200 }                -- Destination Transport Port
+local IPV4_ADDR             = { 1207959553, 2432696319 }  -- Range: 72.0.0.1 - 144.255.255.255
```
Comment (emmericp) [inline examples/ipfix-generation.lua]: can you please change this to parseIPAddress("a.b.c.d")?

**Comment by emmericp** [comment]
Thanks! Looks nice.

But can you please change the indentation to tabs instead of spaces?

**Comment by sosaavedra** [comment]
Glad to contribute. The changes you requested are done.

**Comment by emmericp** [comment]
Thanks!

**Pull request merged**